### ISSUE TITLE
teket.jp/login

### ISSUE THREAD

**Issue by rsuwa** [easylist/easylist#16760]
#### List the website(s) you're having issues with:

`https://teket.jp/login`

#### What happens?

On a popular Japanese e-ticket booking site called teket, a pop-up message appears asking for consent to use cookies.

#### Screenshot(s) 

<https://i.gyazo.com/2d9ffffd5da577ca907b20342e895024.jpg>

#### Filter lists you're using:

```
uBlock Origin 1.50.0:
 since: 30d.22h.39m
Firefox: 102
filterset (summary):
 network: 322598
 cosmetic: 257238
 scriptlet: 43126
 html: 1196
listset (total-discarded, last-updated):
 added:
  adguard-generic: 72756-4440, 2h.57m
  adguard-mobile: 8418-53, 2h.57m
  adguard-spyware-url: 1125-6, 2h.57m
  block-lan: 48-0, 2h.57m
  curben-phishing: 121616-58, 2h.57m
  curben-pup: 189-0, 2h.57m
  adguard-social: 20910-44, 2h.57m
  fanboy-thirdparty_social: 69-3, 2h.57m
  fanboy-cookiemonster: 47740-229, 2h.57m
  fanboy-social: 16105-682, 2h.57m
  ublock-annoyances: 5249-92, 2h.57m
  dpollock-0: 11533-409, 2h.57m
  [12 lists not shown]: [too many]
 default:
  user-filters: 81-2, never
  ublock-filters: 34771-41, 2h.57m
  ublock-badware: 7574-1, 2h.57m
  ublock-privacy: 468-0, 2h.57m
  ublock-unbreak: 2124-1, 2h.57m
  ublock-quick-fixes: 396-46, 2h.57m
  easylist: 67986-1181, 2h.57m
  easyprivacy: 33343-195, 2h.57m
  urlhaus-1: 6452-0, 2h.57m
  plowe-0: 3723-1506, 2h.57m
filterset (user): [array of 84 redacted]
trustedset:
 added: [array of 1 redacted]
switchRuleset:
 added: [array of 197 redacted]
hostRuleset:
 added: [array of 1094 redacted]
urlRuleset:
 added: [array of 4 redacted]
userSettings:
 advancedUserEnabled: true
 largeMediaSize: 1000
hiddenSettings: [none]
supportStats:
 allReadyAfter: 3701 ms
 maxAssetCacheWait: 2500 ms
```

#### Your settings:

- OS/version: macOS Ventura 13.4.1 (c)
- Browser/version: Mullvad Browser 12.5.1
- Adblock extension/version: uBlock Origin 1.50.0

#### Other details:

This is a duplicate of <https://github.com/AdguardTeam/AdguardFilters/issues/155989>, but I thought it should be added to this list, so I created an Issue.

**Comment by Yuki2718** [comment]
This can not be fixed in this repo. There are two types of cookie-consent on the page. The one is automatically displyed if you don't block `googletagmanager.com` (blocked by default on uBO). The [uBO-compatible] rule is for this, but it's already hidden by a generic rule in EasyList Cookie. The other is displayed on clicking Login button (it's not obvious) and addressed by [trusted-set-cookie](https://github.com/AdguardTeam/AdguardFilters/commit/ee95eb959557ee616a3c39e98391a3a325a75341). I'll add the same rule to uBlock filters – Cookie Notices.

**Comment by rsuwa** [comment]
@Yuki2718
Thanks for the quick fix. Is this notation starting with `##+js` a feature of uBO version 1.50.1 or later?
My uBO is version 1.50.0 and it doesn't seem to work.

**Comment by krystian3w** [comment]
In stable: 1.51.0 after allowed by Moz://a Team to global publish: https://github.com/gorhill/uBlock/releases/tag/1.51.0

**Comment by Yuki2718** [comment]
`+js` has long been there. Make sure `uBlock filters – Cookie Notices` is enabled and up-to-date. If not up-to-date, force-update filter lists by Purge cache => Update now. Default update interval is 7 days, so it's likely yours is actually not.

**Comment by rsuwa** [comment]
@Yuki2718 
> `+js` has long been there.

I see, then that does not seem to be a problem.

The filter you mentioned is up to date and the rules you created appear to have been added. However, the cookie confirmation popup still appears. Also, the cookie called `_ul_cookie_consent` does not seem to be set.

Reference:
https://i.gyazo.com/cfea65b4494bea49e46f2d2e5eee46fc.png
https://i.gyazo.com/7ae0a2d0b1060a6d56ec72d9807d7d69.jpg
https://i.gyazo.com/63900e6ff1b0e5d3153c5516d086d77b.jpg

**Comment by krystian3w** [comment]
He use uBo without new scriptlet, enough will be not hide my all post as spam.

**Comment by rsuwa** [comment]
@Yuki2718 
![ubo log](https://i.gyazo.com/fdbba99444956fa57624e95340f296be.jpg)
Here is the log. It looks like your newly added rules are being applied.

**Comment by Yuki2718** [comment]
Sorry, as @krystian3w pointed out `trusted-set-cookie` does not work on `1.50.0`.
@krystian3w TBH I didn' get what you mean.

**Comment by rsuwa** [comment]
@Yuki2718 @krystian3w 
https://github.com/gorhill/uBlock/releases/tag/1.51.0
- Add trusted-set-cookie scriptlet (requires trusted source)

Does this mean that this rule can be applied from 1.51.0?

**Comment by Yuki2718** [comment]
> 
> Does this mean that this rule can be applied from uBO ver.1.51.0?

Right.

**Comment by rsuwa** [comment]
@Yuki2718 @krystian3w 
Thank you both for all you have taught me. My questions have been answered. I will wait patiently for version 1.51.0!

@krystian3w 
By the way, I did not label your post as off-topic or spam.
Rather, I appreciate your contribution.
